recipes/pggb/build.sh

```diff
@@ -6,6 +6,8 @@ n=$((n-1))
 sed -i ${n}'a\if [ $show_version == true ]; then echo "pggb '$PKG_VERSION'"; exit; fi' pggb
 
 mkdir -p $PREFIX/bin
+
 mv pggb $PREFIX/bin
 mv partition-before-pggb $PREFIX/bin
 mv scripts/*.py $PREFIX/bin
+mv scripts $PREFIX/bin
```

Comment:
EOL @AndreaGuarracino. Then this should be good to go. Thanks!

Reply:
Here you are